app/jobs/bulkrax/import_work_job.rb

```diff
@@ -4,20 +4,25 @@ class ImportWorkJob < ApplicationJob
 
     def perform(*args)
       entry = Entry.find(args[0])
+      importer_run = ImporterRun.find(args[1])
```

Comment:
I think this causes problems - ImportRun has to be loaded just in time or multiple jobs will overwrite each other.